src/repoFactory/options/RefreshIdentifierLockOptions.types.ts

```diff
@@ -0,0 +1,4 @@
+export interface RefreshIdentifierLockOptions {
```

Comment:
@ToranSharma It looks like this isn't used?

Reply:
Good spot, thanks 🙂